data/users.json

```
@@ -52,6 +52,27 @@
       "Apple"
     ]
   },
+  {
+    "username": "Ramy Atrouni",
+    "stack": [
+      "React",
+      "Angular",
+      "MySQL",
+      "Express.js",
+      "Sails.js",
+      "ArangoDB",
+      "Redis",
+      "Postgres",
+      "MongoDB",
+      "Vanilla javascript",
```

```suggestion
      "JavaScript",
```

To match `SimpleIcons`